src/readsbml.jl

```diff
@@ -458,7 +458,7 @@ function extract_model(mdl::VPtr)::SBML.Model
 
     initial_assignments = Dict{String,Math}()
     num_ias = ccall(sbml(:Model_getNumInitialAssignments), Cuint, (VPtr,), mdl)
-    for n in 0:(num_ias - 1)
+    for n = 0:(num_ias-1)
```

Comment:
Side note, the fact that you can do for loops with the `for var = range` syntax confuses some new users.  I wish that syntax wasn't allowed at all, and I'm not the only one.

Reply:
ha, this is actually enforced by juliaformatter. Should I complain there?

Reply:
Ugh.  Yes, please, if you're willing to do it :smile:

Reply:
the point is that `in` collides nicely with the `in` operator in a way that is IMO even worse. Why's there even the separator when the parse is unambiguous anyway? :D